codalab/rest/workers.py

```diff
@@ -194,7 +194,7 @@ def code():
     response.set_header("Content-Disposition", 'attachment; filename="code.tar.gz"')
     response.set_header("Content-Type", "application/gzip")
     codalab_cli = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
-    code_dir = os.path.join(codalab_cli, "worker", "codalabworker")
```

Comment:
This whole function is no longer used, so just get rid of it.

Reply:
Hmm, I am actually not sure what was happening here. I double-checked my local branch and also the version that I committed to Github 21 hours ago: https://github.com/codalab/codalab-worksheets/blob/remove-codalabworker-package/codalab/rest/workers.py. None of them have this line. It seems that @epicfaace also observed a similar issue. I'll try to see if I can figure out why >.<...

Reply:
Yeah, remove the whole function.